<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RefactorErl</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="viewMode">
       <enum>QMdiArea::SubWindowView</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuAdd">
     <property name="title">
      <string>Add</string>
     </property>
     <addaction name="actionFile"/>
     <addaction name="actionAppbase"/>
     <addaction name="actionDirectory"/>
     <addaction name="actionInclude"/>
    </widget>
    <addaction name="menuAdd"/>
    <addaction name="actionDrop"/>
    <addaction name="actionSync"/>
    <addaction name="actionReset_database"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuQueries">
    <property name="title">
     <string>Queries</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionQuery_to_run"/>
     <addaction name="actionSkeleton"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionLast_results"/>
    <addaction name="actionRunning_queries"/>
   </widget>
   <widget class="QMenu" name="menuDependency_Graph">
    <property name="title">
     <string>Dependency Graph</string>
    </property>
    <addaction name="actionAll"/>
    <addaction name="actionCircles"/>
    <addaction name="separator"/>
    <addaction name="actionModule"/>
    <addaction name="actionFunction"/>
    <addaction name="actionModule_group"/>
   </widget>
   <widget class="QMenu" name="menuDuplicated_Code">
    <property name="title">
     <string>Duplicated Code</string>
    </property>
    <addaction name="actionNew_duplicated_code_analysis"/>
   </widget>
   <widget class="QMenu" name="menuInvestigations">
    <property name="title">
     <string>Investigations</string>
    </property>
    <addaction name="actionNew_investigation"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionRefactorErl"/>
    <addaction name="actionQt"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuView_Mode">
     <property name="title">
      <string>View Mode</string>
     </property>
     <addaction name="actionSub_Windows"/>
     <addaction name="actionTabs"/>
    </widget>
    <addaction name="actionFile_Browser"/>
    <addaction name="actionQueries"/>
    <addaction name="actionDependency_Graph"/>
    <addaction name="actionDuplicated_Code"/>
    <addaction name="actionInvestigations"/>
    <addaction name="separator"/>
    <addaction name="menuView_Mode"/>
    <addaction name="actionTile_Windows"/>
    <addaction name="actionCascade_Windows"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuQueries"/>
   <addaction name="menuDependency_Graph"/>
   <addaction name="menuDuplicated_Code"/>
   <addaction name="menuInvestigations"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionDirectory">
   <property name="text">
    <string>Directory</string>
   </property>
  </action>
  <action name="actionInclude">
   <property name="text">
    <string>Include</string>
   </property>
  </action>
  <action name="actionAppbase">
   <property name="text">
    <string>Appbase</string>
   </property>
  </action>
  <action name="actionDrop">
   <property name="text">
    <string>Drop</string>
   </property>
  </action>
  <action name="actionSync">
   <property name="text">
    <string>Sync</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRefactorErl">
   <property name="text">
    <string>RefactorErl</string>
   </property>
  </action>
  <action name="actionFile_Browser">
   <property name="text">
    <string>File Browser</string>
   </property>
  </action>
  <action name="actionQueries">
   <property name="text">
    <string>Queries</string>
   </property>
  </action>
  <action name="actionDependency_Graph">
   <property name="text">
    <string>Dependency Graph</string>
   </property>
  </action>
  <action name="actionDuplicated_Code">
   <property name="text">
    <string>Duplicated Code</string>
   </property>
  </action>
  <action name="actionInvestigations">
   <property name="text">
    <string>Investigations</string>
   </property>
  </action>
  <action name="actionQt">
   <property name="text">
    <string>Qt</string>
   </property>
  </action>
  <action name="actionSub_Windows">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Embedded Windows</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionTabs">
   <property name="text">
    <string>Tabs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actionTile_Windows">
   <property name="text">
    <string>Tile Windows</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionCascade_Windows">
   <property name="text">
    <string>Cascade Windows</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionQuery_to_run">
   <property name="text">
    <string>Query to run</string>
   </property>
  </action>
  <action name="actionSkeleton">
   <property name="text">
    <string>Skeleton</string>
   </property>
  </action>
  <action name="actionLast_results">
   <property name="text">
    <string>Last results</string>
   </property>
  </action>
  <action name="actionRunning_queries">
   <property name="text">
    <string>Running queries</string>
   </property>
  </action>
  <action name="actionAll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </action>
  <action name="actionCircles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Circles</string>
   </property>
  </action>
  <action name="actionModule">
   <property name="text">
    <string>Module</string>
   </property>
  </action>
  <action name="actionFunction">
   <property name="text">
    <string>Function</string>
   </property>
  </action>
  <action name="actionModule_group">
   <property name="text">
    <string>Module group</string>
   </property>
  </action>
  <action name="actionNew_duplicated_code_analysis">
   <property name="text">
    <string>New duplicated code analysis</string>
   </property>
  </action>
  <action name="actionNew_investigation">
   <property name="text">
    <string>New investigation</string>
   </property>
  </action>
  <action name="actionReset_database">
   <property name="text">
    <string>Reset database</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
